image_rotate/include/image_rotate/visibility.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
Should be `.hpp`.

Reply:
Not sure if I could rename to .hpp directly, I have just copy-modify the file from https://github.com/ros2/demos/blob/master/composition/include/composition/visibility_control.h